GVFS/GVFS.Common/NuGetUpgrader/ManifestEntry.cs

```diff
@@ -0,0 +1,34 @@
+namespace GVFS.Common.NuGetUpgrader
+{
+    public class ManifestEntry
```

Comment:
This seems a little too generic for me.  I would name it something like `InstallerEntry`, `InstallerData`, `InstallerInfo`?

Reply:
`InstallActionInfo`?